### ISSUE TITLE
README content for algorithms

### ISSUE THREAD

**Issue by likecs** [OpenGenus/cosmos#1073]
Most of the folders contain an README file which is just a copy of one original file containing "cosmos libray".

I feel a small description of the problem or algorithm the code is trying to solve should be present in the README and this should be done for all the folders, even the ones which do not have a README currently. This will improve the overall knowledge base the repository provides and it will be more beneficial to the users who use it.

**Comment by AdiChat** [comment]
Yes, this is required and has been incorporated in a small section of categories.

**Comment by arnavb** [comment]
Can I fix some README's for a small subset of algorithms?
